src/connectComp.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef CONNECTCOMP_H
+#define CONNECTCOMP_H
+
+#include <vector>
+#include <unordered_map>
+#include <string>
+
+namespace found {
+
+struct Point {
+    int x;
+    int y;
+};
+
+struct Edge {
+    int magnitude; // number of points in this edge
+    std::vector<Point> points;
+};
+
+// Function prototype
+std::vector<Edge> NaiveConnectedComponent(unsigned char *image, int imageWidth, int imageHeight);
```

Comment:
Please put a more descriptive function comment here. There should be:
1) General Description of the Funciton
2) Parameter Description
3) Outputs
4) Any modifications on the parameters
5) Preconditions
6) (optional) Postconditions